src/java/main/org/apache/zookeeper/server/command/FourLetterCommands.java

```diff
@@ -151,9 +153,13 @@
      */
     public final static int telnetCloseCmd = 0xfff4fffd;
 
-    public final static HashMap<Integer, String> cmd2String =
+    final static HashMap<Integer, String> cmd2String =
```

Comment:
If we can remove this `public`, then I think we should. Also agree with the consistent `Map` declaration comment.